### PR TITLE
Web speech api

### PR DESCRIPTION
**Pull request by manuscriptmastr**
@niktechnopro Barista can now carry on a (non-intelligent) conversation with user without stopping. I've added the server communication as the next "feature" in Trello, but does this feel like a good "block" for a merge now?

I've commented out the deliver function so the code is available for us to tweak once we do connect it to the server ;)

### CODE REVIEW

File: public/scripts/speech.js [hunk 1]
```diff
@@ -62,25 +78,30 @@ var listen = (phrase) => {
...
 
 var converse = async () => {
-    var inquiry = await listen();
-    await speak('Okay');
-    await converse();
+  while (true) {
+    var reply = await listen();
+    await speak(reply);
+  }
 }
```
Comment (niktechnopro) [inline public/scripts/speech.js]: nice solution with async/await

**Review by niktechnopro — APPROVED** [review]
nice work!!!

**Comment by niktechnopro** [comment]
Very nice work, very clean code and easy to read through it.

File: public/scripts/speech.js (hunk 1 above)
Comment (manuscriptmastr) [reply public/scripts/speech.js]: @nybblr 's idea ;)

**Pull request merged**